tests/Feature/TerminalConsoleModeTest.php

```
@@ -0,0 +1,113 @@
+<?php
+
+use App\Livewire\Project\Shared\Terminal;
+use App\Models\Server;
+
+function consoleModeServer(bool $nonRoot): Server
+{
+    $server = Mockery::mock(Server::class)->makePartial();
+    $server->shouldReceive('isNonRoot')->andReturn($nonRoot);
+
+    return $server;
```

PHPStan (2.2.8)

[warning] Function consoleModeServer() should return App\Models\Server but returns Mockery\LegacyMockInterface.


(return.type)